client/src/pages/GalleryPage.js

```diff
@@ -18,22 +38,88 @@ const GalleryPage = () => {
     doFetch();
   }, []);
 
+  const [currentImage, setCurrentImage] = useState(null);
+  useEffect(() => {
+    if (userData) {
+      setCurrentImage(userData.images[0]);
+    }
+  }, [userData]);
+  console.log("currentImage:", currentImage);
+
+  const [indexOfCurrentImage, setIndexOfCurrentImage] = useState(null);
+  useEffect(() => {
+    if (currentImage) {
+      setIndexOfCurrentImage(userData.images.indexOf(currentImage));
+    }
+  }, [currentImage]);
+  console.log("indexOfCurrentImage:", indexOfCurrentImage);
+  // console.log(userData.images.length);
+
+  // if (userData && currentImage) {
+  //   console.log(userData.images.indexOf(currentImage));
+  //   const indexOfCurrentImage = userData.images.indexOf(currentImage);
+  //   console.log(indexOfCurrentImage + 1);
+  // }
+
+  // console.log(indexOfCurrentImage);
+
+  // const testArray = ["a", "b", "c"];
+  // console.log(testArray.indexOf("b"));
+
+  const nextPlease = () => {
+    const indexOfCurrentImage = userData.images.indexOf(currentImage);
+    setCurrentImage(userData.images[indexOfCurrentImage + 1]);
+  };
+
+  const previousPlease = () => {
+    const indexOfCurrentImage = userData.images.indexOf(currentImage);
+    setCurrentImage(userData.images[indexOfCurrentImage - 1]);
+  };
```

Comment:
Although it is readable, I think you should consider renaming these two variables. Maybe `nextImage` and `previousImage`?